test/distributed/test_c10d.py

```diff
@@ -354,6 +363,31 @@ def test_compare_set(self):
         self.assertEqual(b"new_value0", new_value_result)
         self.assertEqual(b"new_value0", store.get("key0"))
 
+    def test_multi_worker_with_fixed_world_size(self):
+        addr = "localhost"
```

Comment:
nit: Should we make this a standard string constant across the file, like "DEFAULT_HOSTNAME"?